### PR TITLE
Upgrade bot workflow

### PR DESCRIPTION
**Pull request by baptiste-b-pegasys**
### Summary

apply the upgrade get bot with a github action workflow

### Changes*

add a github workflow file, which is a cron job for executing the upgrade geth bot

### References*

- https://github.com/ConsenSys/quorum-bots/pull/1
- https://github.com/ConsenSysQuorum/quorum/issues/191

### CODE REVIEW

File: .github/workflows/upgradebot.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Upgrade geth bot
+on:
+  push:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 6,12 * * *'
```
Comment (baptiste-b-pegasys) [inline .github/workflows/upgradebot.yml:6]:
```suggestion
    - cron:  '0 4 * * *'
```

File: .github/workflows/upgradebot.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: Upgrade geth bot
+on:
+  push:
```
Comment (baptiste-b-pegasys) [inline .github/workflows/upgradebot.yml:3]:
```suggestion
```

File: .github/workflows/upgradebot.yml (hunk 1 above)
Comment (ricardolyn) [reply .github/workflows/upgradebot.yml:6]: yes, I agree that should be simply 4

**Review by ricardolyn — APPROVED** [review]
LGTM 👍

File: .github/workflows/upgradebot.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.15
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.15
+        id: go
+      - name: Clone upgrade geth bot
+        run: |
+          git clone https://github.com/pegasys-dev/quorum-bots.git $GOPATH/src/upgradebot
```
Comment (baptiste-b-pegasys) [inline .github/workflows/upgradebot.yml:24]:
```suggestion
          git clone https://github.com/baptiste-b-pegasys/quorum-bots.git $GOPATH/src/upgradebot
```

**Comment by baptiste-b-pegasys** [comment]
Hmm the problem with git clone and fork projects is that we need to change it at the very end.

File: .github/workflows/upgradebot.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.15
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.15
+        id: go
+      - name: Clone upgrade geth bot
+        run: |
+          git clone https://github.com/baptiste-b-pegasys/quorum-bots.git $GOPATH/src/upgradebot
```
Comment (baptiste-b-pegasys) [inline .github/workflows/upgradebot.yml:24]:
```suggestion
          git clone https://github.com/Consensys/quorum-bots.git $GOPATH/src/upgradebot
```

**Pull request closed without merging**